src/fromager/packagesettings.py

```diff
@@ -142,6 +143,25 @@ def validate_destination_filename(cls, v):
         return v
 
 
+class BuildOptions(pydantic.BaseModel):
+    """Build system options"""
+
+    model_config = MODEL_CONFIG
+
+    cpu_scaling: int = 1
```

Comment:
It sounds like from the description that this is the number of CPU cores to be given to each job, is that right? If so, a name like "cpu_cores_per_job" would make that clearer.

Reply:
I have changed the name to `cpu_cores_per_job` and added a constraint to require a value `>= 1`.